nncf/tensorflow/helpers/model_creation.py

```diff
@@ -62,6 +63,12 @@ def create_compression_algorithm_builder(config: NNCFConfig, should_init: bool)
         CompressionStartedFromConfig(argname="config"),
     ],
 )
+@deprecated(
+    msg="Consider using the 'nncf.quantize()' method instead. "
+    "Please refer to the documentation for guidance on migration.",
+    start_version="2.14.2",
+    end_version="2.14.3",
+)
 def create_compressed_model(
```

Comment:
I suggest to be aligned with PyTorch backend https://github.com/openvinotoolkit/nncf/blob/656605d5f9d054ed9ddcd2299bf6e5ce57896a71/nncf/torch/model_creation.py#L106

Reply:
done